src/main/java/gift/controller/ProductController.java

```diff
@@ -32,8 +34,8 @@ public ResponseEntity<ProductResponseDto> addProduct(@Valid @RequestBody Product
     }
 
     @GetMapping
-    public ResponseEntity<List<ProductResponseDto>> findAllProduct() {
-        return new ResponseEntity<>(productService.findAllProduct(), HttpStatus.OK);
+    public ResponseEntity<Page<ProductResponseDto>> findAllProduct(@PageableDefault(size = 5, sort = "id") Pageable pageable) {
+        return new ResponseEntity<>(productService.findAllProduct(pageable), HttpStatus.OK);
```

Comment:
ResponseEntity의 정적 팩토리 메서드를 활용하면 조금 더 코드를 줄일 수 있을 것 같아요. 😃 

```suggestion
        return ResponseEntity.ok(productService.findAllProduct(pageable));
```